zulipterminal/core.py

```diff
@@ -233,6 +233,12 @@ def show_all_starred(self, button: Any) -> None:
                         anchor=None,
                         starred=True)
 
+    def show_all_mentions(self, button: Any) -> None:
+        # NOTE: Should we ensure we maintain anchor focus here?
```

Comment:
It *seems* to maintain focus so far without? However as with 'starred', perhaps add a note to indicate that, as we may benefit from it later?

Reply:
Added 👍